tgui/packages/tgui/interfaces/NtosCyborgRemoteMonitor.js

```diff
@@ -30,64 +50,134 @@ export const NtosCyborgRemoteMonitorContent = (props, context) => {
   }
 
   return (
-    <>
-      {!card && (
-        <NoticeBox>
-          Certain features require an ID card login.
-        </NoticeBox>
+    <Stack fill vertical>
+      <Stack.Item>
+        <Tabs>
+          <Tabs.Tab
+            icon="robot"
+            lineHeight="23px"
+            selected={tab_main === 1}
+            onClick={() => setTab_main(1)}>
+            Cyborgs
+          </Tabs.Tab>
+          <Tabs.Tab
+            icon="clipboard"
+            lineHeight="23px"
+            selected={tab_main === 2}
+            onClick={() => setTab_main(2)}>
+            Stored Log File
+          </Tabs.Tab>
+        </Tabs>
+      </Stack.Item>
+      {tab_main === 1 && (
+        <>
+          {!card && (
+            <NoticeBox>
```

Comment:
Is missing a stack item around it.

Reply:
Is a `<Stack.Item>` here necessary? It works without it, and putting one in breaks the UI.

Reply:
Yes, breaks how

Reply:
Nevermind, I misunderstood where you wanted the stacks.

This is done now.